Tutorial/Window/co_040000.csv

```diff
@@ -0,0 +1,3 @@
+Title#0,"""Dash Panels"""
+text01#0,"""Press the<%Access> to access the dash panels.<br>This will allow you to dash for a limited time."""
+text02#0,"""While dashing, every time you jump through another dash panel ring,<br>you can dash for longer periods of time.<br>(Re-accessing the dash panels won't be neccessary if you jump through the dash panel rings.）<br><br>(Peformable actions while dashing)<br>Press<%Jump>to jump.<br>Press<%NormalAttack>/<%PhotonArts>to kick.<br><br>The attack power of your kick<br>can be enhanced up to a maximum of 3 stages by jumping through more dash panel rings.<br><br>By skillfully jumping through the dash panel rings, you can quickly reach<br>the destination, and reach the fight immidiately."""
```

Comment:
Noticed misspelling at end of line 3: "immediately"

Reply:
Will fix ASAP.